GitUI/CommandsDialogs/SettingsDialog/Pages/ToolbarSettingsPage.Designer.cs

```diff
@@ -0,0 +1,110 @@
+            this.SuspendLayout();
+            // 
+            // tableLayoutPanelForToolbar
+            // 
+            this.tableLayoutPanelForToolbar.AutoSize = true;
+            this.tableLayoutPanelForToolbar.ColumnCount = 2;
+            this.tableLayoutPanelForToolbar.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
+            this.tableLayoutPanelForToolbar.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
+            this.tableLayoutPanelForToolbar.Controls.Add(this.label2, 0, 0);
+            this.tableLayoutPanelForToolbar.Controls.Add(this.cbBranchOrderingCriteria, 1, 0);
+            this.tableLayoutPanelForToolbar.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.tableLayoutPanelForToolbar.Location = new System.Drawing.Point(0, 0);
+            this.tableLayoutPanelForToolbar.Name = "tableLayoutPanelForToolbar";
+            this.tableLayoutPanelForToolbar.RowCount = 9;
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.tableLayoutPanelForToolbar.Size = new System.Drawing.Size(1333, 684);
+            this.tableLayoutPanelForToolbar.TabIndex = 1;
+            // 
+            // label2
+            // 
+            this.label2.Anchor = System.Windows.Forms.AnchorStyles.Left;
+            this.label2.AutoSize = true;
+            this.label2.Location = new System.Drawing.Point(3, 7);
+            this.label2.Name = "label2";
+            this.label2.Size = new System.Drawing.Size(175, 19);
+            this.label2.TabIndex = 14;
+            this.label2.Text = "Branch ordering criteria";
+            // 
+            // cbBranchOrderingCriteria
+            // 
+            this.cbBranchOrderingCriteria.FormattingEnabled = true;
+            this.cbBranchOrderingCriteria.Items.AddRange(new object[] {
+            "Last access date",
+            "Alphabetically"});
+            this.cbBranchOrderingCriteria.Location = new System.Drawing.Point(184, 3);
+            this.cbBranchOrderingCriteria.Name = "cbBranchOrderingCriteria";
+            this.cbBranchOrderingCriteria.Size = new System.Drawing.Size(213, 27);
+            this.cbBranchOrderingCriteria.TabIndex = 13;
```

Comment:
You have 4 controls on the page yet the tab index is 13.
Please set all indices correctly